### ISSUE TITLE
Object.observe support

### ISSUE THREAD

**Issue by kristfal** [olanod/mapbox-gl-map-element#4]
Hey! Nice wrapper. Got one issue with annotations though: Object.observe is currently only supported in Chrome and Opera.

Using https://github.com/MaxArt2501/object-observe solves the issue in Firefox, but with this shim, Safari (desktop and mobile) doesn't update annotation positions.

Have you found other shims that work in all major browsers, or would you consider dropping Object.observe?

**Comment by olanod** [comment]
Yes, I should definitely use a shim for `Object.observe`. The project I've been working on is a chrome only thing so I didn't need to polyfill that but It's in my plans ;)
